IntegrationTests/SetupFixture.cs

```diff
@@ -42,16 +42,21 @@ public static string GeneratedFileDirectory
 
     private HashSet<string > _copiedFileNames;
 
-    [SetUp]
+    [OneTimeSetUp]
     public void SetUp ()
     {
       s_generatedFileDirectory = Path.Combine (AppDomain.CurrentDomain.BaseDirectory, "GeneratedAssemblies");
       _copiedFileNames = new HashSet<string>();
+
+      // Explicitly set the current directory to the output folder since NUnit 3 has it set to a temporary directory
+      // and some tests break because of it. Tests break because PEVerify cannot find referenced assemblies located
+      // in the output folder.
+      Environment.CurrentDirectory = Path.GetDirectoryName (typeof (SetupFixture).Assembly.Location);
```

Comment:
We always tried to fix this by updating/fixing the code that relies on ambient context instead of just setting EnvDir.  See TestContext.CurrentContext.TestDirectory